File: API/Entities/Organization.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Skillustrator.Api.Entities
{
    public class Organization : EntityBase
    {
        public string Name { get; set; }

        public ICollection<Person> People { get; set; }

    }
}

File: UI/src/app/models/organization.ts
import { Person } from './person';

export class Organization {
  id: number;
  name: string;
  people: Person[];

  constructor(id: number, name: string, people: Person[]) {
    this.id = id;
    this.name = name;
    this.people = people;
  }
}
